sorc/global_cycle.fd/read_write_data.f90

```diff
@@ -472,6 +482,47 @@ subroutine write_data(lensfc,idim,jdim,lsoil, &
    call remove_checksum(ncid, id_var)
  endif
```

Comment:
Since you are writing a new file - gaussian_interp.* - you could have created a new write routine to do this. Just a comment. You don't need to do this.